hardware/bluetooth/bluetooth_hcd.ksy

```diff
@@ -0,0 +1,25 @@
+meta:
+  id: bluetooth_hcd
+  title: Bluetooth Host Controller Interface Control Protocol
+  license: Unlicense
+  file-extension:
+    - hcd
```

Comment:
Quoting from the [KSY Style Guide](https://doc.kaitai.io/ksy_style_guide.html#meta):

> c. **`file-extension`** — if there's **only one** extension, **MUST be a string**; if there are several, (...)

```suggestion
  file-extension: hcd
```